.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ aliases:
   - &defaults
     working_directory: /tmp/storybook
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:8.12.0
```

Comment:
Why?

Reply:
one of the ember dependencies required latest LTS, for some reason the default 8 for circleci is 8.9.4

Reply:
not a ember dependency 

```
error eslint@5.4.0: The engine "node" is incompatible with this module. Expected version "^6.14.0 || ^8.10.0 || >=9.10.0".
error Found incompatible module
```

Reply:
> for some reason the default 8 for circleci is 8.9.4

Weird, maybe it was cached? Let's try setting it back to 8 and see what happens

Reply:
FWIW - Looks like eslint’s engine settings haven’t changed since 5.0.0:

https://github.com/eslint/eslint/commit/b77846d5a4af266d7b5a752fa0d4a403f7177f24

Reply:
I added it back